source/flashback/index.tmpl

```diff
@@ -215,49 +165,6 @@
     </div>
     <!-- END PAGE | SPONSORS -->
 
-    <!-- PAGE | CONTACT -->

```

Comment:
make sure you are up to date with the master branch, i think i already deleted this